.github/workflows/cd-color-stage.yml

```diff
@@ -0,0 +1,54 @@
+name: CD - Color Explorer
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build_linux:
+    runs-on: ubuntu-latest
+
+    env:
+      AZURE_WEBAPP_ACTIVE_STAGE_NAME: color-west-app
+      AZURE_WEBAPP_PASSIVE_STAGE_NAME: color-east-app
+      AZURE_WEBAPP_BUILD_PATH: sites/fast-color-explorer/www
+      AZURE_WEBAPP_SLOT_NAME: stage
+
+    steps:
+    - name: Azure Login
+      uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS_FAST_DESIGN }}
+
+    - name: Checkout Branch
+      uses: actions/checkout@master
+
+    - name: Install Lerna
+      run: sudo yarn global add lerna
```

Comment:
Is `sudo` needed, and does `lerna` need to be installed globally? Lerna can be run in the project directory with `yarn lerna`.

Reply:
@radium-v Pipelines run on a new VM everytime. If there is a way to improve performance  I'm game.  We've been doing this in most of our pipelines internally/externally.  At one point it was failing without `sudo`.  You'll see on windows/macos it's not needed and eliminated.